code/src/index.css

```diff
@@ -11,3 +11,78 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+
+button {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  margin-top:15px;
+  margin-bottom: 15px;;
+  padding: 0.6em 1.2em;
+  font-size: 16px;
+  color: #000;
+  background-color: rgb(251, 171, 171);
+  border: none;
+  border-radius: 50px;
+  transition: all 0.3s ease 0s;
+  cursor: pointer;
+  outline: none;
+  font-size: 1rem;
+  padding: 0.7rem 0.9rem;
+}
+
+button:hover{
+  background-color: rgb(247, 132, 132);
+}
+ 
+
+.thought-text {
+  font-family: 'Inconsolata', monospace;
+  font-weight: 700;
+}
```

Comment:
To make the long messages line-break, I added `overflow: auto` here. Like we talked about this week and on our Monday meeting, there are several ways of doing it though :)